openhtf/core/station_api.py

```diff
@@ -561,6 +579,11 @@ def list_tests(self):
     _LOG.debug('RPC:list_tests() -> %s results', len(retval))
     return retval
 
+  def get_phase_descriptors(self, test_uid):
+    """Get phase descriptors for the given test UID, or None if invalid UID."""
+    phases = openhtf.Test.from_uid(test_uid).descriptor.phases
+    return [{'name': phase.name, 'doc': phase.doc} for phase in phases]
```

Comment:
why not return the phase descriptors themselves? They're not that heavyweight and the information can be useful to the frontend, such as measurement names